qaframework-bdd-tests/src/test/resources/org/openmrs/uitestframework/test.properties

```diff
@@ -3,5 +3,5 @@ login.username=admin
 login.password=Admin123
 login.location=Pharmacy
 webdriver=chrome
-headless=true
+headless=false
```

Comment:
Ensure you dont push headless to false, after making your changes, make this to true then push .

Reply:
thanks recommitted after returning headless to true